backend/request-forwarding.js

```diff
@@ -7,12 +7,7 @@ const extractRequestPaginationMiddleware = (req, res, next) => {
   next()
 }
 
-const setPagingHeaders = (context, res) => {
```

Comment:
we no longer need to request forward from react, so will all be removed when react stuff disappears